keyboards/handwired/onekey/keymaps/backlight/keymap.c

```diff
@@ -0,0 +1,26 @@
+#include QMK_KEYBOARD_H
+
+//Tap Dance Declarations
+enum {
+  TD_BL = 0
+};
+
+void dance_cln_finished (qk_tap_dance_state_t *state, void *user_data) {
+  // noop
+}
+
+void dance_cln_reset (qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count == 1) {
+    backlight_step();
```

Comment:
so, 2 taps for backlight toggle? 3 or more to put in breathing mode? comment would be helpful

Reply:
Updated to switch with comments